main/conductor_main.cpp

```diff
@@ -1,13 +1,12 @@
 #include <PPN-microbench/conductor.hpp>
-#include <PPN-microbench/ops/flops.hpp>
-#include <PPN-microbench/ops/iops.hpp>
+#include <PPN-microbench/memory.hpp>
+
 
 int main() {
 
     Conductor conductor;
 
-    conductor.addBench(new Flops(5))
-        .addBench(new Iops(5))
+    conductor.addBench(new Memory("Memory",1))
```

Comment:
I think you can declare them raw in `memory.cpp` and let the constructor empty here

You can check my 2 other comments in `memory.cpp` and `memory.hpp` to complete this

```c++
conductor.addBench(new Memory())
```

Reply:
Thank you, but it won't let me let this empty. It has to match with the instance of Microbench.